Floofbot/Modules/Utilities.cs

```diff
@@ -255,5 +257,127 @@ public async Task ServerList()
                 Trash = true
             }, true);
         }
+        [Command("convert")]
+        [Alias("conv")]
+        [Summary("Converts units to other units, such as Celcius to Fahrenheit.")]
+
+        public async Task convert(string input)
```

Comment:
Remember a parameter description and also use [Remainder] to take the whole rest of the command, refer to other commands in utilities for this